connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java

```diff
@@ -55,11 +59,12 @@ public void translate(MovePlayerPacket packet, GeyserSession session) {
             session.getUpstream().sendPacketImmediately(moveEntityBack);
             return;
         }
-
-        if (!isValidMove(session, packet.getMode(), entity.getPosition(), packet.getPosition())) {
-            session.getConnector().getLogger().info("Recalculating position...");
-            recalculatePosition(session, entity, entity.getPosition());
-            return;
+        if (((GeyserConnector)Geyser.getConnector()).getConfig().isMovementCheck()) {
```

Comment:
This should be replaced with `session.getConnector().getConfig().isMovementCheck()`